specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json

```diff
@@ -0,0 +1,2684 @@
+        },
+        "privateIPAddress": {
+          "description": "The virtual network private IP address of the IP configuration.",
+          "type": "string"
+        }
+      }
+    },
+    "LoadBalancerFrontendIPConfiguration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the resource that is unique within the set of frontend IP configurations used by the load balancer. This name can be used to access the resource.",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LoadBalancerFrontendIPConfigurationProperties",
+          "description": "Properties of load balancer frontend ip configuration."
+        }
+      },
+      "required": [
+        "name",
+        "properties"
+      ]
+    },
+    "LoadBalancerConfigurationProperties": {
+      "type": "object",
+      "properties": {
+        "frontendIPConfigurations": {
+          "description": "Specifies the frontend IP to be used for the load balancer. Only IPv4 frontend IP address is supported. Each load balancer configuration must have exactly one frontend IP configuration.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LoadBalancerFrontendIPConfiguration"
+          }
+        }
+      },
+      "required": [
+        "frontendIPConfigurations"
+      ]
+    },
+    "LoadBalancerConfiguration": {
+      "description": "Describes the load balancer configuration.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource Id",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the Load balancer",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LoadBalancerConfigurationProperties",
+          "description": "Properties of the load balancer configuration."
+        }
+      },
+      "required": [
+        "name",
+        "properties"
+      ]
+    },
+    "CloudServiceNetworkProfile": {
+      "description": "Network Profile for the cloud service.",
+      "type": "object",
+      "properties": {
+        "loadBalancerConfigurations": {
+          "description": "List of Load balancer configurations. Cloud service can have up to two load balancer configurations, corresponding to a Public Load Balancer and an Internal Load Balancer.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LoadBalancerConfiguration"
+          }
+        },
+        "swappableCloudService": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The id reference of the cloud service containing the target IP with which the subject cloud service can perform a swap. This property cannot be updated once it is set. The swappable cloud service referred by this id must be present otherwise an error will be thrown."
```

Comment:
Update the description to add the bold text:   "description": "The id reference of the cloud service containing the target IP with which the subject cloud service can perform a swap. This property cannot be updated **or deleted** once it is set. The swappable cloud service referred by this id must be present otherwise an error will be thrown."
        }